### ISSUE TITLE
Vite build never ends

### ISSUE THREAD

**Issue by xSorc** [Subwaytime/vite-aliases#23]
### Describe the bug
When we import `vite-aliases` package(v0.7.1) inside `vite.config.ts` - `vite build` never ends. Process don't die

### Reproduction
To reproduce this you need to create simple vite app, include `vite-aliases` package inside `vite.config.ts` and run `vite build`

here is a link to simple repo: https://github.com/xSorc/test-vite-aliases

### System Info

Output of `npx envinfo --system --npmPackages vite,@vitejs/plugin-vue,vite-aliases --binaries --browsers`:

```node
  System:
    OS: Windows 10 10.0.19042
    CPU: (6) ia32 Intel(R) Core(TM) i5-8600K CPU @ 3.60GHz
    Memory: 2.64 GB / 15.94 GB
  Binaries:
    Node: 14.17.0 - C:\Program Files (x86)\nodejs\node.EXE
    Yarn: 1.22.10 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.14.13 - C:\Program Files (x86)\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.423.0), Chromium (91.0.864.48)
    Internet Explorer: 11.0.19041.1
  npmPackages:
    vite: 2.3.7 => 2.3.7
    vite-aliases: 0.7.1 => 0.7.1
```

**Comment by Subwaytime** [comment]
Hey there!
Thanks for reporting that Issue!

Vite 0.7.1/0.7.2 are still in `WIP` so if you need vite-aliases now, consider using 0.6.3 for the time beeing.

Other than that, the issue itself is caused by chokidar, because it runs on build to, ive already been working on a fix!
Will be included in `0.7.3`

**Comment by Subwaytime** [comment]
fixed in `0.8.0`
